src/NServiceBus.Core/UnitOfWork/UnitOfWorkBehavior.cs

```diff
@@ -42,7 +47,7 @@ class UnitOfWorkBehavior : Behavior<PhysicalMessageProcessingContext>
             }
         }
 
-        List<Exception> AppendEndExceptionsAndRethrow(Exception initialException)
+        List<Exception> AppendEndExceptionsAndRethrow(Stack<IManageUnitsOfWork> unitsOfWork, Exception initialException)
```

Comment:
I fixed it better:
![f0804c28-9359-11e5-8f78-da9bc4722f3f](https://cloud.githubusercontent.com/assets/601206/11392545/243986b6-9363-11e5-93d9-5e7fa1024f72.png)